### PR TITLE
ISPN-6926 Server hangs on start

### PR DESCRIPTION
**Pull request by gustavonalle**
https://issues.jboss.org/browse/ISPN-6926

Problem solved in my environment, let's see on TC

### CODE REVIEW

File: server/core/src/main/java/org/infinispan/server/core/AbstractProtocolServer.java [hunk 1]
```diff
@@ -35,6 +36,21 @@
    protected A configuration;
    private ObjectName transportObjName;
    private MBeanServer mbeanServer;
+   private static final String UseEpollProperty = "infinispan.server.channel.epoll";
+   private static final boolean IsLinux = System.getProperty("os.name").toLowerCase().startsWith("linux");
+   private static final boolean EpollDisabled = System.getProperty(UseEpollProperty, "true").equalsIgnoreCase("false");
+   private static boolean useNativeEpoll;
```
Comment (wburns) [inline server/core/src/main/java/org/infinispan/server/core/AbstractProtocolServer.java]: Can this be final as well?

Comment (gustavonalle) [inline server/core/src/main/java/org/infinispan/server/core/AbstractProtocolServer.java]: yes

**Comment by gustavonalle** [comment]
Updated @wburns

**Comment by wburns** [comment]
Seems fine to me, pulling.

**Comment by wburns** [comment]
Integrated into master, thanks @gustavonalle !

**Pull request closed without merging**